<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid rgb(100, 169,139);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QToolButton" name="min_btn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="最小化"/>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover
{
	background-color:rgba(200, 200, 200,85);
}
QToolButton:pressed
{
	background-color:rgba(220,220,220,85);
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>540</width>
        <height>260</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>260</height>
       </size>
      </property>
      <widget class="QtMaterialTextField" name="account">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>19</y>
         <width>300</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>40</height>
        </size>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QtMaterialTextField" name="password">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>79</y>
         <width>300</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>300</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <widget class="QCheckBox" name="auto_login">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{
color:grey;
}</string>
        </property>
        <property name="text">
         <string>自动登录</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="remenber_password">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{
color:grey;
}</string>
        </property>
        <property name="text">
         <string>记住密码</string>
        </property>
       </widget>
       <widget class="QPushButton" name="find_password">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>0</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border:none;
color:grey}
QPushButton:hover{
color:black;
}</string>
        </property>
        <property name="text">
         <string>找回密码</string>
        </property>
       </widget>
      </widget>
      <widget class="QtMaterialRaisedButton" name="login">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>300</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Uighur</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="make_account">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:none;
color:grey;
}
QPushButton:hover{
color:black;
}</string>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
      <widget class="ElaComboBox" name="account_box">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>301</width>
         <height>0</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
background:transparent;
}
QComboBox::down-arrow {
image: url(://photo/xiajiantou.png); /* 默认箭头 */
}
QComboBox:active::down-arrow {
image: url(://photo/shangjiantou.png); /* 聚焦时的箭头 */
}</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="down_btn">
       <property name="geometry">
        <rect>
         <x>415</x>
         <y>19</y>
         <width>15</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#down_btn{
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>account_box</zorder>
      <zorder>account</zorder>
      <zorder>password</zorder>
      <zorder>widget_2</zorder>
      <zorder>login</zorder>
      <zorder>make_account</zorder>
      <zorder>down_btn</zorder>
     </widget>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="head_photo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>76</width>
        <height>76</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>76</width>
        <height>76</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>76</width>
        <height>76</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QToolButton" name="close_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="关闭"/>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover 
{
	background-color:rgb(255, 0, 0);
}
QToolButton:pressed
{
	background-color:rgb(170, 0, 0);
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElaComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">elacombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialtextfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
